test/e2e/docker_test.go

```diff
@@ -17,14 +17,14 @@ import (
 )
 
 // Labels
-func TestDockerKubernetes125Labels(t *testing.T) {
+func TestDockerKubernetesLabels(t *testing.T) {
```

Comment:
Are we not renaming this to 126Labels because there is the only test for Docker labels?

Reply:
Yeah exactly & we always just update it to the latest version